src/main/kotlin/HbaseClient.kt

```diff
@@ -59,19 +58,19 @@ open class HbaseClient(val connection: Connection, val columnFamily: ByteArray,
 
         if (!namespaces.contains(namespace)) {
             try {
-                logger.info("Creating namespace '$namespace'.")
+                logger.info("Creating namespace", "namespace", namespace)
                 connection.admin.createNamespace(NamespaceDescriptor.create(namespace).build())
             }
             catch (e: NamespaceExistException) {
-                logger.info("'$namespace' already exists, probably created by another process")
+                logger.info("Namespace already exists, probably created by another process", "namespace", namespace)
             }
             finally {
                 namespaces[namespace] = true
             }
         }
 
         if (!tables.contains(tableName)) {
-            logger.info("Creating table '$dataTableName'.")
+            logger.info("Creating table", "dataTableName", "$dataTableName")
```

Comment:
_name fits how we are doing the key names in the other apps?

Reply:
changed